docs/developer-docs/latest/setup-deployment-guides/installation/docker.md

```diff
@@ -1,169 +1,454 @@
 ---
-title: Install from Docker - Strapi Developer Docs
-description: Quickly create a Strapi application using our official Strapi Docker images.
+title: Custom Docker Container - Strapi Developer Docs
+description: Quickly create a Docker container from a local project.
 canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/installation/docker.html
 ---
 
-# Installing using Docker
+# Running Strapi in a Docker container
 
 :::caution
-This Docker image is only for Strapi v3. For now, Strapi will not update the image for v4.
-However, to build an image compatible with Strapi v4, we recommend following [this guide](https://blog.dehlin.dev/docker-with-strapi-v4) by Simen Daehlin, Community Star at Strapi.
+Strapi does not build any official container images. The following instructions are provided as a courtesy to the community. If you have any questions, please reach out to the community on [Discord](https://discord.strapi.io).
+:::
+
+The following documentation will guide you through the process of building a custom [Docker](https://www.docker.com/) container with an existing Strapi project.
+
+Docker is an open platform that allows to develop, ship and run applications by using containers (i.e. packages containing all the parts an application needs to function, such as libraries and dependencies). Containers are isolated from each other and bundle their own software, libraries and configuration files; they can communicate with each other through well-defined channels.
+
+## Prerequisites
```

Comment:
WDYT about changing this to the prerequisites callout and then deleting line 19?